rust/connlib/snownet/src/node.rs

```diff
@@ -353,6 +367,20 @@ where
         let packet_start = 4;
         let packet_end = 4 + packet_len;
 
+        let socket = match conn.state {
+            ConnectionState::Connected { peer_socket, .. } => peer_socket,
+            ConnectionState::Connecting {
+                ref mut buffered, ..
+            } => {
+                tracing::debug!("No socket has been nominated yet, buffering packet");
```

Comment:
Might be helpful to print packet dest or at least packet length here so we don't get duplicated messages we can't act upon.